src/components/Button/index.js

```diff
@@ -1,52 +1,87 @@
 import React from 'react';
 import cn from 'classnames';
+
 export const BUTTON_OPTIONS = {
   PADDING: {
-    ZERO: false, // no padding at all
-    DEFAULT: 'default', // default padding == 2rem
     SMALL: 'small', // for footer section
-    BIG: 'big', // big padding == 8rem,
-    MOB_PADDING: 'mobile_padding'
+    BIG: 'big' // big padding == 8rem,
   },
-  WIDTH: {
-    SMALL: 'small', // 8rem
-    BIG: 'big' //16rem
+  backgroundColor: {
+    C100: 'blue',
+    C200: 'red',
+    C300: 'black1',
+    C400: 'black2',
+    C500: 'grey1',
+    C600: 'grey2',
+    C700: 'grey3',
+    C800: 'grey4',
+    C900: 'off white',
+    C1000: 'green',
+    WHITE: 'white'
   },
-  HIEGHT: {
-    SMALL: 'small', // 2.5rem
-    BIG: 'big' // 3rem
+  COLOR: {
+    C100: 'blue',
+    C200: 'red',
+    C300: 'black1',
+    C400: 'black2',
+    C500: 'grey1',
+    C600: 'grey2',
+    C700: 'grey3',
+    C800: 'grey4',
+    C900: 'off white',
+    C1000: 'green',
+    WHITE: 'white'
   }
 };
 
 export const Button = ({
   children,
   rounded,
-  bgColor,
+  backgroundColor,
+  color,
   shadow,
-  width,
-  icon,
   padding,
-  height,
   opacity,
   className
 }) => {
   return (
     <button
       className={cn(
         'flex items-center justify-center outline-none focus:outline-none',
-        { 'shadow-md': shadow },
+        { 'shadow-outline': shadow },
         { 'bg-opacity-50': opacity },
         { 'rounded-lg': rounded },
```

Comment:
Use property names that reflect their use. So, names like: isRounded, isTransparent would be more appropriate.